app/assets/javascripts/show-free-text-question.js

```diff
@@ -0,0 +1,31 @@
+window.GOVUKAdmin = window.GOVUKAdmin || {};
+window.GOVUKAdmin.Modules = window.GOVUKAdmin.Modules || {};
+
+(function (Modules) {
+  "use strict";
+
+  Modules.ShowFreeTextQuestion = function () {
+    this.start = function($element) {
+      function addEventListener() {
+        var $yesRadio = $element.find('.yes-radio');
```

Comment:
Could you please [use `.js`-prefixed classes](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/docs/javascript.md#use-js--prefixed-classes-for-interaction-hooks)?